ghost/i18n/lib/i18n.js

```diff
@@ -2,15 +2,19 @@ const i18next = require('i18next');
 
 const SUPPORTED_LOCALES = [
     'af', // Afrikaans
+    'ar', // Arabic
     'bg', // Bulgarian
     'bs', // Bosnian
     'ca', // Catalan
     'cs', // Czech
     'da', // Danish
     'de', // German
+    'de-CH', // Swiss German
```

Comment:
not 100% sure - but do these need to be all lowercase? (also ln46)

Reply:
Looks to me like we're following https://en.wikipedia.org/wiki/IETF_language_tag.  Whether that was an explicit decision between multiple conventions or not, I don't know.

The codes that are new to the file are consistent with pt-BR (Portuguese as spoken in Brazil), which has been there for quite a while.  While we could decide to change conventions, that would require changing names of directories, and some sort of migration strategy for sites with pt-BR, which has been there in that form for quite some time.

So, no, they don't, and let's not make work for ourselves. :)  Re sr-Cyrl (and zh-Hant, not new), that's correct.  Script designators are Titlecase, while country designators are UPPERCASE, and language codes are lowercase.